src/Glpi/Controller/Form/SubmitAnswerController.php

```diff
@@ -76,16 +76,28 @@ public function __invoke(Request $request): Response
         $form = $this->loadSubmittedForm($request);
         $this->checkFormAccessPolicies($form, $request);
 
-        $answers = $this->saveSubmittedAnswers($form, $request);
-        $links = $answers->getLinksToCreatedItems();
+        try {
+            $answers = $this->saveSubmittedAnswers($form, $request);
+            $links = $answers->getLinksToCreatedItems();
 
-        if ($is_unauthenticated_user) {
-            AltchaManager::getInstance()->removeChallenge($altcha);
-        }
+            if ($is_unauthenticated_user) {
+                AltchaManager::getInstance()->removeChallenge($altcha);
+            }
 
-        return new JsonResponse([
-            'links_to_created_items' => $links,
-        ]);
+            return new JsonResponse([
+                'links_to_created_items' => $links,
+            ]);
+        } catch (\Throwable $th) {
+            $messages = [];
+            if (isset($_SESSION['MESSAGE_AFTER_REDIRECT'][ERROR])) {
+                $messages = $_SESSION['MESSAGE_AFTER_REDIRECT'][ERROR];
+                unset($_SESSION['MESSAGE_AFTER_REDIRECT'][ERROR]);
+            }
+
+            return new JsonResponse([
+                'errors' => $messages,
+            ], Response::HTTP_INTERNAL_SERVER_ERROR);
```

Comment:
`HTTP_INTERNAL_SERVER_ERROR` should not be used if the response correspond to an invalid input, is it possible to differenciate error cases here?